Readme.md

```diff
@@ -1,23 +1,51 @@
+
+![Label updated on grid callback](updated-label-on-grid-callback.png)
+
+[ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) sends callbacks to update row data on the server. The control raises the [EndCallback](https://docs.devexpress.com/AspNet/js-ASPxClientGridView.EndCallback) client event after a callback is successfully executed. You can handle the event to perform custom actions with another control when a callback is finished.
 
-<p>By default, the ASPxGridView works during callbacks and there's no way to update an external control (that isn't a child control of the callback owner) on the server side. The following article describes this limitation in detail:</p><p><a href="https://www.devexpress.com/Support/Center/p/K18387">The Concept of Callbacks</a></p><p>However, the ASPxGridView as any other ASP.NET control has the <a href="http://documentation.devexpress.com/#AspNet/DevExpressWebASPxGridViewASPxGridView_JSPropertiestopic">JSProperties</a> feature that allows passing a value from the  server to the client. Also, the client-side EndCallback is raised each time when a callback is executed successfully.<br />
-So, it is possible to set a JSProperty on the server, get it on the <a href="http://documentation.devexpress.com/#AspNet/DevExpressWebASPxGridViewScriptsASPxClientGridView_EndCallbacktopic">EndCallback</a> and change a "target" control using its client-side capabilities.</p><p>This example illustrates how to use the ASPxLabel to identify that the grid was successfully updated. </p><p>A similar approach my be used with any other DevExpress ASP.NET control that can send callbacks (ASPxCallback, ASPxCallbackPanel, ASPxTreeList, ASPxComboBox and so on).</p>
+```aspx
+<dx:ASPxGridView ...>
+    <ClientSideEvents EndCallback="OnEndCallBack" />
+</dx:ASPxGridView>
+```
 
-<br/>
+The [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) implements the [JSProperties](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.JSProperties) property that allows you to pass information from the server to the client. In this example, an updated row’s key is saved to the [JSProperties](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBase.JSProperties) property on the server and then this value is used on the client to display information about the updated row.
+
+```cs
+protected void Page_Load(object sender, EventArgs e) {
+    ASPxGridView1.JSProperties["cpIsUpdated"] = "";
+}
+protected void ASPxGridView1_RowUpdated(object sender, DevExpress.Web.Data.ASPxDataUpdatedEventArgs e) {
+    if (e.Exception == null) {
+        ((ASPxGridView)sender).JSProperties["cpIsUpdated"] = e.Keys[0];
+    }
+}
+```
+
+```js
+function OnEndCallBack(s, e) {
+    if (s.cpIsUpdated != '') {
+        clientLabel.SetText('The category ' + s.cpIsUpdated + ' is updated successfully');
```

Comment:
Can we remove the article ("The") in this message?

Reply:
```suggestion
        clientLabel.SetText('Category ' + s.cpIsUpdated + ' is updated successfully');
```